cmake/README.md

```diff
@@ -160,7 +189,9 @@ cmake -DTBB_VALGRIND_MEMCHECK=ON ..
 make -j memcheck-<test name> # or memcheck-all to scan all tests 
 ```
 
-## Test specification (Doxygen)
+## Test specification
+
+Use Doxygen to test oneTBB specification:
```

Comment:
```suggestion
Use Doxygen to generate oneTBB test specification:
```

"Test" in the chapter name is a noun but not a verb :)
Feel free to update it somehow if it is confusing.